java/java.editor.base/src/org/netbeans/modules/java/editor/base/semantic/TokenList.java

```diff
@@ -240,6 +240,30 @@ public void run() {
         });
     }
     
+    public Token currentToken() {
+        Token[] res = new Token[1];
+        doc.render(new Runnable() {
```

Comment:
Jackpot:
warning: This anonymous inner class creation can be turned into a lambda expression.